pkg/controller/controller_utils.go

```diff
@@ -18,6 +18,7 @@ package controller
 
```

Comment:
Fixed. There are 2 instances of replication left (podControl.CreateReplica takes a replication controller as input) . However, I think podControl belongs outside of controller/replication, because it can be used by other controllers (in fact I'll be using podControl in daemon).